### ISSUE TITLE
[4.0] Registration: links privacy / terms / conditions

### ISSUE THREAD

**Issue by Kostelano** [joomla/joomla-cms#33664]
### Steps to reproduce the issue
Log in to the control panel, in the settings of the USERS component, allow registration. Also activate the site login module for the frontend.

Also include any plugins that provide terms of use / rules / privacy.

**Content - Confirm Consent**
**System - Privacy Consent**
**User - Profile** (TOS parameters)
**User - Terms and Conditions**

Try to register a new user in the frontend. Note that you have NOT configured plugins. You haven't created a single article or menu item. But when registering, you can see the lines "Read the full privacy policy" / "Read the full terms and conditions". These are not links, but just text.

![Screenshot_1](https://user-images.githubusercontent.com/8440661/117551212-4f0aa480-b04d-11eb-8465-0af34f809745.png)

![Screenshot_2](https://user-images.githubusercontent.com/8440661/117551214-4fa33b00-b04d-11eb-8dfc-0a1aa58474b4.png)

In my opinion, they should be inaccessible to the eyes of a new user until you have configured plugins and created separate pages for rules / conditions, etc. In this case, he should only see the abbreviated text, which is a constant.

Also, I will mention an issue that I posed a long time ago for version 3.9 - it still hasn't been resolved at the moment.

https://github.com/joomla/joomla-cms/issues/28537

**Comment by drmenzelit** [comment]
I would like to add here that if you have configured a link for privacy policy and/or for terms and conditions the semantic of the label (containing a link) is wrong, WAVE marks the label as "orphaned form label"

Very nice article about inputs and labels: https://css-tricks.com/html-inputs-and-labels-a-love-story/#dont-put-interactive-elements-inside-labels

**Comment by brianteeman** [comment]
This should be marked as a release blocker

**Comment by alikon** [comment]
please test #34537
